### PR TITLE
nikhil's impl of DR student search

### PR DESCRIPTION
**Pull request by nikhiljain12**
Please accept my changes.

### CODE REVIEW

File: Selenium/selenium-dhammareg.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        try:
+            self.driver.find_element_by_xpath(xpath)
+        except NoSuchElementException:
+            return False
+        return True
+
+    def openDR(self):
+        self.driver.get(self.BASE_URL)
+        self.driver.implicitly_wait(5)
+        uname = self.driver.find_element_by_xpath("//*[@id='LoginForm_username']")
```
Comment (ShubhraJain) [inline Selenium/selenium-dhammareg.py]: Why didn't you just you find_element_by_id ??

File: Selenium/selenium-dhammareg.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        uname.send_keys("nikhilj")
+        passw = self.driver.find_element_by_xpath("//*[@id='passwordField']")
```
Comment (ShubhraJain) [inline Selenium/selenium-dhammareg.py]: same here.

File: Selenium/selenium-dhammareg.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        adv.click()
+        cityInput = self.driver.find_element_by_xpath("//input[@id='Students_City']")
+        cityInput.clear()
+        cityInput.send_keys(city)
+        stateInput = self.driver.find_element_by_xpath("//input[@id='state']")
+        stateInput.clear()
+        stateInput.send_keys('CA')
+        
+        num_3 = self.searchNumSat(">2")
+        num_5 = self.searchNumSat(">4")
```
Comment (ShubhraJain) [inline Selenium/selenium-dhammareg.py]: nice how you have implemented functions to search for >2 and >4

File: Selenium/selenium-dhammareg.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    driver = webdriver.Firefox()
+    dr = DhammaReg(driver)
+    dr.openDR()
+    cities = ['san juan bautista', 'salinas', 'river oaks', 'prunedale', 'hollister', 'elkhorn']
+    for city in cities:
+        print(city, "  ", dr.advancedStudentSearch(city))
+
+    driver.close()
+
+visitAllLinksSlow()
```
Comment (ShubhraJain) [inline Selenium/selenium-dhammareg.py]: Does it matter if there is no end of line in python file? github shows me a warning about it.

File: Selenium/selenium-dhammareg.py (hunk 1 above)
Comment (nikhiljain12) [inline Selenium/selenium-dhammareg.py]: you're right. Not really necessary to use xpath here.

File: Selenium/selenium-dhammareg.py (hunk 2 above)
Comment (nikhiljain12) [inline Selenium/selenium-dhammareg.py]: same here. ;)

File: Selenium/selenium-dhammareg.py (hunk 3 above)
Comment (nikhiljain12) [inline Selenium/selenium-dhammareg.py]: thanks :)

File: Selenium/selenium-dhammareg.py (hunk 4 above)
Comment (nikhiljain12) [inline Selenium/selenium-dhammareg.py]: should be fine I guess.

**Pull request merged**